### ISSUE TITLE
Non-JQuery solution

### ISSUE THREAD

**Issue by ekkis** [CodersPress/jQuery-Currency#6]
since you've figured out all the logic needed to make this happen, it would be really much more useful to have a javascript function that simply received a value and converted it:

FormatCurrency("2300"); // $2,300.00

...rather than having to have an object on the page to do it.  keeping your current implementation is, of course, very useful but in many other cases I may not have an element on the page I want to convert, I just have a value.

in other words, exposing the helper methods would give the module much greater functionality.

my two pence.

**Comment by mikbe** [comment]
I second this. I don't want to be setting a field twice, I just want to convert a float to currency then I'll set the field.

**Comment by three3211** [comment]
:+1:

**Comment by ekkis** [comment]
I'm not sure whether I issued a pull request but for whoever cares, I forked and patched here: https://github.com/ekkis/jQuery-Currency
